carbon-design-system/carbon
# feat(grid): add span support to Column and grid package

Closes #10473

This PR adds a snippet to our migration guide for the changes coming to Grid. It also adds in support for `start`, `end` to our grid package and `Column` component. It documents this behavior in the `Grid Start End` story in storybook.

#### Changelog

**New**

- Add `col-end-{i}` support to the grid package

**Changed**

- Update `col-start-{i}` to iterate through all grid lines (number of columns + 1)
- Update `Column` to support `start`, `end` in breakpoint props
- Update tests for Column to test for combinations

**Removed**

#### Testing / Reviewing

- Review migration material and verify its accuracy
- View the "Grid Start End" story and verify the offsets work correctly
- Make sure changes have corresponding tests
